aioesphomeapi/api.proto

```diff
@@ -610,7 +623,7 @@ message SubscribeLogsResponse {
   option (no_delay) = false;
 
   LogLevel level = 1;
-  bytes message = 3;
+  string message = 3;
```

Comment:
This is a breaking change - but super convenient having the name coerced to a string.